src/db/migrations/18-add-anotherblock.ts

```diff
@@ -0,0 +1,48 @@
+
+import { Knex } from 'knex';
+
+import { NftFactory, NFTContractTypeName, NFTStandard } from '../../types/nft';
+import { MusicPlatform, MusicPlatformType } from '../../types/platform';
+import { addNftFactory, addPlatform, removeNftFactory, removePlatform } from '../migration-helpers';
+
+const ANOTHERBLOCK_PLATFORM: MusicPlatform = {
+  id: 'anotherblock',
+  type: MusicPlatformType['single-track-multiprint-contract'],
+  name: 'anotherblock',
+}
+
+const ANOTHERBLOCK: NftFactory = {
+  address: '0x9ef75b412d8466b9b35f3a1bf7a809a5c6d0aa7c',
+  startingBlock: '15320924',
+  platformId: ANOTHERBLOCK_PLATFORM.id,
+  contractType: NFTContractTypeName.default,
+  standard: NFTStandard.ERC721,
+  typeMetadata: {
+    overrides: {
+      track: {
+        websiteUrl: 'https://anotherblock.io/weekend-on-a-tuesday',
+        title: 'Weekend on a Tuesday',
+        lossyAudioIPFSHash: 'QmYDoCuj8wcSSr5okZXLfc5eFQNDdNy7sBTQWQYts9vrRy', 
+        lossyArtworkIPFSHash: 'QmTovjG2VAV9DCSVzHqvHe8PFtUoUQ5FUYDBv2qZWHpazC',
+        lossyArtworkURL: 'https://spinamp.mypinata.cloud/ipfs/QmTovjG2VAV9DCSVzHqvHe8PFtUoUQ5FUYDBv2qZWHpazC'
+      },
+      artist: {
+        name: 'r3hab & laidback luke',
+        artistId: 'r3hab & laidback luke',
+        platformInternalId: 'r3hab & laidback luke',
+        avatarUrl: 'https://themusicessentials.com/wp-content/uploads/2022/08/R3HAB-WEEKEND-ON-A-TUESDAY-e1660897451989-1068x756.jpg.webp',
+        websiteUrl: 'https://www.anotherblock.io/',
```

Comment:
this is the website for the whole platform, not just the artist? shouldn't it be just for them? or at least to one of their drops